apps/judicial-system/xrd-api/jest.config.ts

```diff
@@ -8,6 +8,8 @@ export default {
   coverageDirectory: '<rootDir>/coverage/apps/judicial-system/xrd-api',
   displayName: 'judicial-system-xrd-api',
   testEnvironment: 'node',
+  detectOpenHandles: false,
+  detectLeaks: false,
```

Comment:
Is this debug code or meant for production?

Reply:
Meant to be like this. We want all our tests suits to have this config because otherwise we cannot run tests. Note that is is not production config, it is jest test config.